### PR TITLE
Added particular support for the Roccat Elo 7.1 Air.

### PR DESCRIPTION
**Pull request by lukasgr90**
These changes add the new device "Roccat Elo 7.1 Air". For now only control of the LEDs is supported. More might come.

Partially implements #127

### CODE REVIEW

File: src/devices/roccat_elo_7_1_air.c [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ */
+static int send_receive(hid_device* hid_device, const unsigned char* data, 
+			int size, unsigned char* out_buffer) 
+{
+    struct timespec ts;	
+    unsigned char send_buffer[64];
+    int r;
+
+    memset(send_buffer, 0, sizeof(send_buffer));
+    memcpy(send_buffer, data, size);
```
Comment (Sapd) [inline src/devices/roccat_elo_7_1_air.c:49]: Thank you!

Just a question regarding your send_receive function. As I dont quite understand the sense.

E.g. when you do this:
```
     unsigned char cmd92[64] = {0xff, 0x02};

     hid_write(hid_device, cmd92, sizeof(cmd92)/sizeof(cmd92[0]));
```

The first line, will automatically pad everything with 0s. Thats defined in the C-Standard. So you dont actually have to clear the array with memset beforehand.

Comment (lukasgr90) [reply src/devices/roccat_elo_7_1_air.c:49]: Thank you for the feedback!

If that's defined in the C-Standard, I'll of course remove this as obsolete.

It is quite a time ago that I actively programmed C / Assembler and just remembered, that I should rather be defensive about such things. ;-)

File: src/devices/roccat_elo_7_1_air.c [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+static const uint16_t PRODUCT_ID = 0x3a37;
+
+static int elo71Air_switch_lights(hid_device* hid_device, uint8_t on);
+static int elo71Air_send_inactive_time(hid_device* hid_device, uint8_t num);
+
+void elo71Air_init(struct device** device)
+{
+    device_elo71Air.idVendor = VENDOR_ROCCAT;
+    device_elo71Air.idProductsSupported = &PRODUCT_ID;
+    device_elo71Air.numIdProducts = 2;
```
Comment (lukasgr90) [inline src/devices/roccat_elo_7_1_air.c:19]: After looking at usage of this parameter at other device's code I think that I did not set it correctly!?

@Sapd Can you please have a look and give a hint, how it is used properly (if necessary)?

File: src/devices/roccat_elo_7_1_air.c (hunk 1 above)
Comment (lukasgr90) [reply src/devices/roccat_elo_7_1_air.c]: I hope that I corrected the code properly.

**Review by lukasgr90 — COMMENTED** [review]
Thanks to @iss0 for contributing the udev rules!

File: src/devices/roccat_elo_7_1_air.c (hunk 2 above)
Comment (Sapd) [reply src/devices/roccat_elo_7_1_air.c:19]: Yep, it must be 1.
It simply says, how big the array device_elo71Air.idProductsSupported is. As you only provide one id, its size is 1.
When you specify 2, the loop in device_manager.c would run into some other memory and possibly crash or do some other unexpected things.

**Pull request merged**